src/Cli.Tests/EndToEndTests.cs

```diff
@@ -768,4 +768,23 @@ public void TestBaseRouteIsConfigurableForSWA(string authProvider, bool isExcept
             Assert.AreEqual("/base-route", runtimeConfig.Runtime.BaseRoute);
         }
     }
+
+    /// <summary>
+    /// Test to validate that whenever the flag rest.request-body-strict is included in the init command,
+    /// the runtimeconfig is initialized such that the runtime allows extraneous fields in the request body for REST.
+    /// </summary>
+    /// <param name="isRequestBodyStrict">Value of the rest.request-body-strict flag in the init command.</param>
+    [DataTestMethod]
+    [DataRow(false, DisplayName = "REST request body allows extraneous fields.")]
+    [DataRow(true, DisplayName = "REST request body doesn't allow extraneous fields.")]
+    public void TestRestRequestBodyStrictMode(bool isRequestBodyStrict)
```

Comment:
can you add an additional test method to validate the behavior when you do not supply the switch `--rest.request-body-strict`? This will help validate backwards compat.

Reply:
added